src/config/airflow.cfg

```diff
@@ -63,7 +63,9 @@ sql_alchemy_schema =
 parallelism = 32
 
 # The number of task instances allowed to run concurrently by the scheduler
```

Comment:
Should this comment be removed seeing as the line of code that followed it has been removed?

Reply:
:+1: